.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -1,10 +1,13 @@
 name: Validate Gradle Wrapper
-on: [push, pull_request]
+on: pull_request
```

Comment:
Actually having also

```
on:
  push:
    branches:
      - develop
	  - release
```

Would be useful

Reply:
The idea is that we do have a run just after the merge on the `develop` and `release` branch so we're sure the HEAD is green.

Reply:
That was my next PR)

Reply:
Great, so do you want to merge this?

Reply:
No, I just needed to get to the laptop to update this PR.
Now it is ready

Reply:
Great, feel free to merge as soon as it's ✅